www/js/key.js

```diff
@@ -0,0 +1,8 @@
+if (AWS && AWS.config) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["AWS" is not defined.](https://www.codacy.com/app/warlot-pierre-quentin/MGL7130/file/2415286640/issues/source?bid=3191644&fileBranchId=3235078#l1)